test/evergreen.yml

```diff
@@ -1455,12 +1456,6 @@ tasks:
         vars:
           GNU_C_VERSION: -DGNU_C_VERSION=9
           GNU_CXX_VERSION: -DGNU_CXX_VERSION=9
-      # Check that WiredTiger will compile for code coverage, as this may switch to alternative
-      # implementations of some code which may generate different compiler errors to a 'normal' build
-      - func: "compile wiredtiger"
-        vars:
-          CMAKE_BUILD_TYPE: -DCMAKE_BUILD_TYPE=Coverage
-          CODE_COVERAGE_FLAGS: -DCODE_COVERAGE_MEASUREMENT=1 -DINLINE_FUNCTIONS_INSTEAD_OF_MACROS=1
```

Comment:
I've moved this compile task into the new `compile-uncommon-build-flags` task. I doesn't really make sense for it to be part of the "compile gcc versions" job.